recipe/build.sh

```diff
@@ -29,8 +29,15 @@ for shared in OFF ON; do
         -DBUILD_SHARED_LIBS=$shared
 
     make -j${CPU_COUNT}
+
     # ctest seems to have weird PATH assumptions
     export PATH=$PWD/EXAMPLE:$PWD/TEST:$PATH
-    ctest
+    # avoid heavy oversubscription of resources: already 1-2 MPI ranks (processes)
+    export OMP_NUM_THREADS=1
+    export KMP_NUM_THREADS=${OMP_NUM_THREADS}
+    export MKL_NUM_THREADS=${OMP_NUM_THREADS}
+    export CTEST_REGEX="-R (pdtest_1x1|pdtest_1x2|pdtest_2x1|pddrive)"
```

Comment:
Can you add a comment here why this is used and also remove it for x86_64 ?

Reply:
Same comment for the same block: the tests are NxM processes that MPI spawns and the limitation is true for all CI we are using.

Reply:
Wouldn't `OMP_NUM_THREADS=$CPU_COUNT` work then?
Also, this would remove the testing for the case when there's oversubscription. oversubscribing shouldn't fail (except when it takes too much memory or other resources).

Reply:
No, because it's `mpirun -n <N>` and this number gets multiplied with the number of threads in `CPU_COUNT`. We cannot reliably test oversubscription unless someone likes to click restart whenever the watch dog daemons caught us.